client/src/main.ts

```diff
@@ -8,10 +8,14 @@ import KeyCloakService from 'sbc-common-components/src/services/keycloak.service
 
 // NB: order matters - do not change
 import 'quill/dist/quill.core.css'
-import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader
+import '@mdi/font/css/materialdesignicons.min.css'
 import '@/sass/main.sass'
 import '@/sass/overrides.sass'
 
+import '@/assets/scss/base.scss'
+import '@/assets/scss/layout.scss'
+import '@/assets/scss/overrides.scss'
```

Comment:
When I got here, some components were using SASS and others were using SCSS.

For consistency with the other Entities UIs, I went with SCSS.

We should migrate SASS &rarr; SCSS at some point in the future.